experiment/clustersecretbackup/main.go

```diff
@@ -84,6 +86,8 @@ func gatherOptions(fs *flag.FlagSet, args ...string) options {
 	fs.StringToStringVar(&o.secrets, "secret-name", nil, "namespace:name of secrets to be backed up, in the form of --secret-name=<namespace>=<name>. By default all secrets in the chosen namespace(s) are backed up.")
 	fs.BoolVar(&o.update, "update", false, "Controls whether update existing secret or not, if false then secret will only be created")
 	fs.BoolVar(&o.dryRun, "dryrun", false, "Controls whether this is dry run or not")
+	fs.BoolVar(&o.skipServiceAccount, "skip-sa", false, "Controls whether to skip service account tokens")
```

Comment:
I feel this should be default to true, I'm not seeing any reason storing these k8s generated tokens